tests/pages/account.test.tsx

```diff
@@ -0,0 +1,228 @@
+          status: "validated",
+          type: ReceivedIdentityTypes.PHONE,
+          value: "0622116652",
+        },
+      ],
+      emailIdentities: [
+        {
+          id: "8f79dcc1-530b-4982-878d-33f0def6a7cf",
+          primary: false,
+          status: "validated",
+          type: ReceivedIdentityTypes.EMAIL,
+          value: "test@test.test",
+        },
+      ],
+    };
+
+    const component = mount(
+      <ThemeProvider theme={lightTheme}>
+        <GlobalStyle />
+        <Layout>
+          <Account sortedIdentities={mockedSortedResponse} />
+        </Layout>
+      </ThemeProvider>,
+    );
+
+    const emailTrash = component.find({ type: IdentityTypes.EMAIL });
+    const phoneTrash = component.find({ type: IdentityTypes.PHONE });
+
+    expect(emailTrash).toHaveLength(0);
+    expect(phoneTrash).toHaveLength(2);
+  });
+
+  test("it should appear for both email and phone if phone and email list > 1", () => {
+    const mockedSortedResponse: SortedIdentities = {
+      phoneIdentities: [
+        {
+          id: "8f79dcc1-530b-4982-878d-33f0def6a7cf",
+          primary: false,
+          status: "validated",
+          type: ReceivedIdentityTypes.PHONE,
+          value: "0622116655",
+        },
+        {
+          id: "8f79dcc1-530b-4982-878d-33f0def6a7cf2",
+          primary: false,
+          status: "validated",
+          type: ReceivedIdentityTypes.PHONE,
+          value: "0622116652",
+        },
+      ],
+      emailIdentities: [
+        {
+          id: "8f79dcc1-530b-4982-878d-33f0def6a7cf",
+          primary: false,
+          status: "validated",
+          type: ReceivedIdentityTypes.EMAIL,
+          value: "test@test.test",
+        },
+        {
+          id: "8f79dcc1-530b-4982-878d-33f0def6a7cf2",
+          primary: false,
+          status: "validated",
+          type: ReceivedIdentityTypes.EMAIL,
+          value: "test2@test.test",
+        },
+      ],
+    };
+    const component = mount(
+      <ThemeProvider theme={lightTheme}>
+        <GlobalStyle />
+        <Layout>
+          <Account sortedIdentities={mockedSortedResponse} />
+        </Layout>
+      </ThemeProvider>,
+    );
+
+    const emailTrash = component.find({ type: IdentityTypes.EMAIL });
+    const phoneTrash = component.find({ type: IdentityTypes.PHONE });
+
+    expect(emailTrash).toHaveLength(2);
+    expect(phoneTrash).toHaveLength(2);
+  });
+
+  test("it should appear for neither email nor phone if phone and email list = 0 (a specific messsage should appear instead)", () => {
+    const mockedSortedResponse: SortedIdentities = {
+      phoneIdentities: [],
+      emailIdentities: [],
+    };
+    const component = mount(
+      <ThemeProvider theme={lightTheme}>
+        <GlobalStyle />
+        <Layout>
+          <Account sortedIdentities={mockedSortedResponse} />
+        </Layout>
+      </ThemeProvider>,
+    );
+
+    const emailTrash = component.find({ type: IdentityTypes.EMAIL });
```

Comment:
Thinking about it, could we make a find more precise and only search the component we're looking for and not only one of its props? (I'm thinking searching both the component and its prop)

Reply:
Unfortunately this is the most precise I could be, because giving the component an `id` or `className` throws the following error:
`Property 'className' does not exist on type 'IntrinsicAttributes & FetchIconButtonProps & { children?: ReactNode; }'.ts(2322)`

Reply:
What about something like `component.find(FetchIconButton).find({ type: IdentityTypes.EMAIL })`?

Reply:
OK fixed! 6697393